### DISCUSSION TITLE
Upload all notes in a folder

### DISCUSSION THREAD

**Discussion by stoweboyd** [alangrainger/share-note#49] [Vote for upcoming features]
I maintain folders for many purposes, one of which include a 'knowledge base' of notes for various people, concepts, and companies. It would be great if I could execute a sharing of all the notes in a folder,  instead of manually sharing one by one.

**Comment by beesinblenders** [comment]
If anyone needs a quick workaround, here's a Templater script that does the trick (delay might be too short if your notes take a long time to parse):  https://github.com/alangrainger/share-note/discussions/116#discussioncomment-10640197

**Comment by aiproductguy** [comment]
This would work nicely with the Folder Note plugin and convention, which should be adopted by Obsidian other markdown tools as a convention that gets us a step closer to Web-local formatting semantics.

obsidian://show-plugin?id=folder-note-plugin
https://github.com/xpgo/obsidian-folder-note-plugin

I think content-addressible diffing needs to be a prereq to make this efficient without compromising Share.Note's genius simplicity.

**Comment by alangrainger** [comment]
Share Note is intended for quick, non-permanent note sharing.

For publishing a folder or for making any sort of long-term documentation, I suggest [looking at Quartz](https://github.com/jackyzha0/quartz), and my companion [Quartz Publisher plugin](https://github.com/alangrainger/quartz-publish).

Quartz is what I use to publish folders from my vault as different websites - the [Share Note documentation](https://docs.note.sx) is published with Quartz.
